src/components/BootProtoSelector.js

```diff
@@ -24,15 +24,23 @@ import bootProtocol from '../lib/model/bootProtocol';
 
 import { FormSelect, FormSelectOption } from '@patternfly/react-core';
 
-const bootProtocolOptions = bootProtocol.values.map(bootProto => {
+// FIXME: These are the boot protocols supported by wicked. Improve the selector to choose them
+// dynamically according to the network backend in use.
+const supportedBootProtocols = [
+    bootProtocol.NONE,
```

Comment:
But the ones supported by wicked are the bootProtocol.values from:

https://github.com/dgdavid/cockpit-wicked/blob/master/src/lib/model/bootProtocol.js#L35

Isn't it?

Reply:
@imobachgs :point_up_2:  :point_up_2: